src/rust/python-parser/cxx-bridge.h

```diff
@@ -0,0 +1,5 @@
+#include "kj/string.h"
```

Comment:
Nit: In KJ-style, includes to paths that depend on include directives (i.e. are not a relative path from the current directory) should use angled brackets. (As opposed to Google style, where using quotes is preferred).